frontend/src/scenes/authentication/signup/SignupContainer.tsx

```diff
@@ -0,0 +1,58 @@
+import { useValues } from 'kea'
+import { preflightLogic } from 'scenes/PreflightCheck/preflightLogic'
+// import { signupLogic } from './signupLogic'
+import { userLogic } from 'scenes/userLogic'
+import { SceneExport } from 'scenes/sceneTypes'
+import { BridgePage } from 'lib/components/BridgePage/BridgePage'
+// import RegionSelect from '../../RegionSelect'
```

Comment:
I thought you had added the CTA to move to cloud EU/US below the logo, but I don't see it

edit: I see the <RegionSelect is still there in the form, this is just a commented import, would just remove it from here 👌

Reply:
<RegionSelect is also used on the Login, btw

Reply:
I'll be removing the region select in a subsequent pull. I cleaned this up but you'll still see the region select for now until my next PR.